### PR TITLE
transition to resources.yml

### PR DESCRIPTION
**Pull request by timjore**
The base pieces are in place for consistent and concise explanation of all resources (content, tools, training).

### CODE REVIEW

File: _data/resources.yml [hunk 1]
```diff
@@ -175,93 +198,163 @@
...
 # TOOLS: Translation
 #
 
-# translationStudio (tS)
+# Autographa
+- name: Autographa
+  abbreviation:
+  category: tools
+  sub-category: translation
+  url: https://unfoldingword.org/autographa/
```
Comment (jag3773) [inline _data/resources.yml]: This page doesn't exist.

File: _data/resources.yml [hunk 2]
```diff
@@ -175,93 +198,163 @@
...
 
 
-# translationManager (tM)
+# tM
 - name: translationManager
   abbreviation: tM
-  category: tool
+  category: tools
   sub-category: translation
   url: https://unfoldingword.org/tm/
```
Comment (jag3773) [inline _data/resources.yml]: This page doesn't exist

**Review by jag3773 — CHANGES_REQUESTED** [review]
All of the URLs need to be specified as relative paths.  For instance, "https://unfoldingword.org/tq/" should be "/tq/".

Also, the pages that don't exist need to be created before this can be merged.

File: _data/resources.yml [hunk 3]
```diff
@@ -175,93 +198,163 @@
...
   why: "provides free, remote storage and collaboration services for creators and translators of biblical content."
 
-
+#
 # TOOLS: Publishing
+#
 
-#### Door43 Live Site (D43)
+# D43
 - name: translationDatabse
```
Comment (jag3773) [inline _data/resources.yml]: This should be "Door43 Live Site", not "translationDatabase".

File: _data/resources.yml [hunk 4]
```diff
@@ -175,93 +198,163 @@
...
-  icon: https://unfoldingword.org/assets/img/icon-td.png
-  what: "A way to request that GL translations are ready to become source texts for OLs."
-  why: "allows translations to become source texts in the Door43 platform."
-
+# STR
+#- name: Source Text Request Form
+#  abbreviation: STR
+#  category: tool
+#  sub-category: research
+#  url: https://td.unfoldingword.org
```
Comment (jag3773) [inline _data/resources.yml]: This URL should be http://ufw.io/pub .

File: _data/resources.yml [hunk 5]
```diff
@@ -175,93 +198,163 @@
...
+#  icon: https://unfoldingword.org/assets/img/icon-td.png
```
Comment (jag3773) [inline _data/resources.yml]: No icon for this, or a D43 icon.

File: _data/resources.yml (hunk 1 above)
Comment (timjore) [reply _data/resources.yml]: @jag3773 am I correct that the links on the pages themselves should still be built like this: 

`href="{{ '/tq/' | prepend: site.baseurl }}"`

**Review by jag3773 — APPROVED** [review]

**Pull request merged**